_releaser/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# syntax=docker/dockerfile:1
+
+ARG GO_VERSION=1.18
+
+FROM scratch AS publishdir
+
+FROM golang:${GO_VERSION}-alpine AS base
+RUN apk add --no-cache jq openssl
+ENV CGO_ENABLED=0
+WORKDIR /src
+
+FROM base AS vendor
+RUN --mount=type=bind,target=.,rw \
+  --mount=type=cache,target=/go/pkg/mod \
+  go mod tidy && go mod download
+
+FROM base AS releaser
+RUN --mount=type=bind,target=. \
```

Comment:
I wonder if the mount will invalidate cache if anything changes, and if it would be better to copy just the go mod and go sum

Reply:
Hum yeah might be better indeed